backend/src/main/java/wooteco/retrospective/config/LoginInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package wooteco.retrospective.config;
+
+import org.springframework.http.HttpMethod;
+import org.springframework.web.servlet.HandlerInterceptor;
+import wooteco.retrospective.exception.AuthorizationException;
+import wooteco.retrospective.infrastructure.auth.AuthorizationExtractor;
+import wooteco.retrospective.infrastructure.auth.JwtTokenProvider;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginInterceptor implements HandlerInterceptor {
+
+    private JwtTokenProvider jwtTokenProvider;
+
+    public LoginInterceptor(JwtTokenProvider jwtTokenProvider) {
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+
+        if (request.getMethod().equals(HttpMethod.OPTIONS.name())) {
+            return true;
+        }
+
+        String accessToken = request.getHeader("Authorization");
```

Comment:
```suggestion
        String accessToken = request.getHeaderHttpHeaders.AUTHORIZATION);
```

제공하는 상수를 사용하는게 버그를 줄이는 길이라고 생각합니다.

Reply:
이런게 있는지 처음 알았네요...!